src/authorization.js

```diff
@@ -415,16 +413,16 @@ class Authorization {
     if (this.virtual) {
       return []
     }
-    var statement
-    var fragment = rdf.namedNode('#' + this.hashFragment())
-    var ns = vocab(rdf)
-    var statements = [
+    let statement
+    let fragment = rdf.namedNode('#' + this.hashFragment())
```

Comment:
Shouldn't NamedNode use full IRI instead of just fragment? Just as PermissionSet constructor accepts `aclUrl` Authorization should also accept one and use it to construct NamedNode with using IRI